spec/views/admin/submissions/show.html.erb_spec.rb

```diff
@@ -177,6 +178,26 @@
       expect(@submission.reload.deleted_at).to be_nil
     end
 
+    context 'notes' do
+      before do
+        @admin = Fabricate(:user, email: "admin@art.sy")
+        3.times { |i| Fabricate(:note, submission: @submission, created_by: @admin.id, body: "Note #{i + 1}") }
+        page.visit "/admin/submissions/#{@submission.id}"
+      end
+
+      it 'shows a list of notes' do
+        expect(page).to have_content("Notes")
+        within(:css, '.notes-section') do
+          expect(page).to have_content('admin@art.sy', count: 3)
+          expect(page).to have_content('Note 1')
+          expect(page).to have_content('Note 2')
+          expect(page).to have_content('Note 3')
```

Comment:
A bit rusty on my capybara, but this seemed to work.

Reply:
Have you tried something like:

```rb
expect(page).to have_content('Note 1\nNote 2\nNote 3')
```

Reply:
This actually doesn't work because there is an author section on each note.